benchmark/run.js

```diff
@@ -6,7 +6,7 @@ const CLI = require('./_cli.js');
 
 const cli = CLI(`usage: ./node run.js [options] [--] <category> ...
   Run each benchmark in the <category> directory a single time, more than one
-  <categoty> directory can be specified.
+  <category> directory can be specified.
```

Comment:
Given that it's in an entirely different file, I'd prefer that it get its own PR, yeah. (But I wouldn't stop this from being approved over it.)